token-lending/program/src/state/reserve.rs

```diff
@@ -545,12 +611,12 @@ pub struct ReserveConfig {
 
 /// Additional fee information on a reserve
 ///
-/// These exist separately from interest accrual fees, and are specifically for
-/// the program owner and frontend host.  The fees are paid out as a percentage
-/// of collateral token amounts during repayments and liquidations.
+/// These exist separately from interest accrual fees, and are specifically for the program owner
+/// and frontend host. The fees are paid out as a percentage of liquidity token amounts during
+/// repayments and liquidations.
 #[derive(Clone, Copy, Debug, Default, PartialEq)]
 pub struct ReserveFees {
-    /// Fee assessed on `BorrowReserveLiquidity`, expressed as a Wad.
+    /// Fee assessed on `BorrowObligationLiquidity`, expressed as a Wad.
```

Comment:
I prefer the original name because it reads "borrow liquidity from the reserve". This reads to me like "borrow liquidity from an obligation" which doesn't make sense

Reply:
I agree that this name isn't precisely right for the abstraction.

What I wanted to express is that the names of all instructions should present a consistent target for their API. In the case of borrow/repay/liquidate, you're acting upon an obligation and not just upon some reserve(s), and the instructions require an initialized obligation with deposits.

If this distinction doesn't matter, then we can go back to `BorrowReserveLiquidity`, `RepayReserveLiquidity`, and `LiquidateObligation`, but I think something is also lost this way. I'll see if I can think of a better way to express this idea.

Reply:
Maybe just shorten everything by focusing not on what is being acted upon but what is being done by the caller with it.

For reserves: `DepositLiquidity`, `RedeemCollateral`

For obligations: `DepositCollateral`, `WithdrawCollateral`, `BorrowLiquidity`, `RepayLiquidity`, `Liquidate`

What do you think?

Reply:
Cool, that makes sense to organize instructions by what they're acting on

Reply:
So, just to be clear, leave it as I have it now?

Reply:
Ah yeah, the way you organized it is great, but the enum name as `BorrowObligationLiquidity` is still weird to me

Reply:
Yeah, I don't love it. I might end up changing it if I think of something better while working on the frontend.

Reply:
Todo added, marking resolved for now.